CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ Please follow the recommendations outlined at [keepachangelog.com](http://keepac
 Changes since last non-beta release.
 
 *Please add entries here for your pull requests that are not yet released.*
+### Fixed
+- Fixed the bug in `bin/dev` and `bin/dev-static` scripts [PR 1519](https://github.com/shakacode/react_on_rails/pull/1519) by [ahangarha](https://github.com/ahangarha).
```

Comment:
"Fixed the bug" is way too vague for changelog entries.

Would "switch from exec to system for bin script calls to shell" be more accurate?

Reply:
How is it now?